pkg/kapp/config/config.go

```diff
@@ -52,6 +52,7 @@ type WaitRuleConditionMatcher struct {
 	Failure                    bool
 	Success                    bool
 	SupportsObservedGeneration bool
+	SupportsUnblockingChanges  bool
```

Comment:
Open to suggestions on choosing a better name for this. (Other alternative I am inclined towards is `unblockBlockedChanges`)

Reply:
Might be missing context here, when did we move away from "non-blocking"

Reply:
I closed that PR sometime back as it wasn't aligning with the ask on issue. [This](https://github.com/vmware-tanzu/carvel-kapp/issues/427#issuecomment-1092355922) comment might help in getting to know more about the need.